MonicasFlagToC.user.js

```diff
@@ -1174,9 +1174,9 @@ function initQuestionPage()
             reviews: []
          };
          
-         var flagList = Array.from(dom.querySelectorAll(".post-timeline .event-rows tr[data-eventtype=flag]:not(.deleted-event-details)"));
-         var flaggedCommentList = Array.from(dom.querySelectorAll(".post-timeline .event-rows tr[data-eventtype=comment] td.event-comment .toggle-comment-flags-container a[data-flag-ids]"));
-         var reviewList = Array.from(dom.querySelectorAll(".post-timeline .event-rows tr[data-eventtype=review]:not(.deleted-event-details)"));
+         var flagList = Array.from(dom.querySelectorAll(".post-timeline .event-rows tr[data-eventtype=flag]"));
+         var flaggedCommentList = Array.from(dom.querySelectorAll(".post-timeline .event-rows tr[data-eventtype=comment] td.event-comment .js-toggle-comment-flags[data-flag-ids]"));
+         var reviewList = Array.from(dom.querySelectorAll(".post-timeline .event-rows tr[data-eventtype=review]"));

```

Comment:
This changes the nature of the query - it'll now pull back two rows for the completed event. Pretty sure the same applies for flags, although that one you might get away with as there's a sanity-check that skips flags which seem to be missing critical data. 

Should be able to extract this function and run it against a few timelines with completed & pending reviews & flags, then verify that the data which is generated actually looks sane; save some headaches on down the road.